coffee-shop-application/src/main/kotlin/com/motycka/edu/order/OrderItemRepositoryImpl.kt

```diff
@@ -0,0 +1,32 @@
+package com.motycka.edu.order
+
+import org.jetbrains.exposed.sql.Database
+import org.jetbrains.exposed.sql.transactions.transaction
+
+class OrderItemRepositoryImpl(private val database: Database) : OrderItemRepository {
```

Comment:
You chose to pass database reference explicitly, which is not required in such simple application but it actually is a pretty good idea as it provides flexibility and control. The downside is that it makes the code just slightly more complex (not much), and code complexities accumulate. But it is a good practice in general.

If we want this level of flexibility in our app, it would be good idea to also introduce dependency injection framework (such as Koin).